src/twisted/conch/ssh/transport.py

```diff
@@ -257,6 +259,49 @@ def makeMAC(self, seqid, data):
         data = struct.pack(">L", seqid) + data
         return hmac.HMAC(self.outMAC.key, data, self.outMAC[0]).digest()
 
+    def createPacket(self, payload: bytes, outgoingPacketSequence: int) -> bytes:
```

Comment:
I moved this in ciphers as I feel that it can help reduce the API for SSHCiphers.

But I am ok to move it back to SSHTransportBase